lavamoat/browserify/main/policy.json

```diff
@@ -4455,6 +4455,62 @@
         "react": true
       }
     },
+    "react-devtools-core": {
```

Comment:
The policy changes are appearing because there is a manual ignore missing after this line: https://github.com/MetaMask/metamask-extension/blob/168bdf44a5fd30c72d9743939583657a7eefb73d/development/build/scripts.js#L1033

We should add `react-devtools-core` as well to keep it out of the bundle on non-dev builds. No tree-shaking to rely on with the old build.